profiles/forms.py

```diff
@@ -1,16 +1,15 @@
 from django import forms
-from .models import Ape
+from .models import Ape, HealthRecord
 
 
 class ProfileForm(forms.ModelForm):
     class Meta:
-        model = Ape
-        fields = [
+        ape_model = Ape
+        ape_fields = [
             "name",
             "age",
```

Comment:
[thought] I just realized we could store the Ape's DoB instead and compute the age when needed. That way it is always correct!

Reply:
Yeah I had that thought too ha. Def the way to do it, need to look it up